Source/Svg/Svg.csproj

```diff
@@ -108,21 +111,15 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' Or '$(TargetFramework)' == 'netcoreapp3.1' Or '$(TargetFramework)' == 'net5.0'">
-    <PackageReference Include="System.Drawing.Common" Version="5.0.0" />
     <PackageReference Include="System.ObjectModel" Version="4.3.0" />
   </ItemGroup>
 
```

Comment:
These are not needed I think as System.ObjectModel seems to be in the Microsoft.NETCore.App framework reference.

Reply:
And what about other target frameworks ?

Reply:
I am not sure if it's in netcoreapp2.1's and netcoreapp3.1's but netcoreapp2.1 is *technically* EOL so I guess it could be removed from Svg.NET so that way all .NET Core 2.1 applications would need to migrate to a supported framework version.